paddle/framework/framework.proto

```diff
@@ -46,6 +47,7 @@ message OpDesc {
     repeated bool bools = 11;
     optional int32 block_idx = 12;
     optional int64 l = 13;
+    repeated int32 block_idxs = 14;
```

Comment:
what's the relation between `block_idx` and `block_idxs`?

can they merge into one field?  given `optional = repeated[size=0+]`

Reply:
block_idxs = vector<block_idx>

Reply:
I will take a look if they can be merged